xapian-core/queryparser/cjk-tokenizer.h

```diff
@@ -36,6 +36,11 @@
 #pragma GCC diagnostic push
 #pragma GCC diagnostic ignored "-Wold-style-cast"
 
+// work around missing U_PLATFORM_HAS_WINUWP_API on libicu MacOSX builds.
+#ifndef U_PLATFORM_HAS_WINUWP_API
+#define U_PLATFORM_HAS_WINUWP_API 0
+#endif
+
 #include <unicode/unistr.h>
 #include <unicode/brkiter.h>
```

Comment:
Generally we sort groups of includes (unless ICU requires this order, in which case "ick").

Reply:
Fixed in a858fcb